src/components/Header.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from "react";
+
+const Header = () => {
+    const [dispalayDarkMode, setDispalayDarkMode] = useState(false);
+
+    const handleDarkModeClick = () => {
+        setDispalayDarkMode(!dispalayDarkMode)
+    }
+
+    useEffect(() => {
+        if (dispalayDarkMode) {
+            document.body.classList.add("dark-theme");
+        } else {
+            document.body.classList.remove("dark-theme");
+        }
```

Comment:
Refer to [this](https://stackoverflow.com/a/23663302/8274756). 

You can simplify this by doing this : 

```
useEffect(() => {
    document.body.classList.toggle("dark-theme", displayDarkMode);
}, [displayDarkMode]);
```